### PR TITLE
Add tests for Loader

### LINKED ISSUE #96: (SP: 1) Write unit test for "Loader" component
### Component unit test
Unit test for "**Loader**" component
Scenaries descriptions:
- [ ] Component should be in the document

[Link to component](https://github.com/Space2Study-UA-1156/Client-01/blob/develop/src/components/loader/Loader.jsx)

Current coverage:
image

### PR DESCRIPTION
**Pull request by khytrykv**
image
image

### CODE REVIEW

**Review by holus-bolus — APPROVED** [review]

**Comment by YaroslavChuiko** [comment]
I think the path should be `tests/unit/components/loader/Loader.spec.jsx` same as the path to the Loader component
image

**Comment by khytrykv** [comment]
> I think the path should be `tests/unit/components/loader/Loader.spec.jsx` same as the path to the Loader component image

Fixed

**Review by YaroslavChuiko — APPROVED** [review]

**Review by voloshynvv — APPROVED** [review]

File: tests/unit/components/loader/Loader.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/components/loader/Loader.spec.jsx:1]: Mock the `userService` hook

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: tests/unit/components/loader/Loader.spec.jsx
Comment (khytrykv) [reply tests/unit/components/loader/Loader.spec.jsx:1]: But we don't use the `userService` in this component. Why should I mock it?

Comment (vlad-khrychov) [reply tests/unit/components/loader/Loader.spec.jsx:1]: Yes, sorry, my mistake

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**